var/spack/repos/builtin/packages/omnitrace/package.py

```diff
@@ -13,13 +13,21 @@ class Omnitrace(CMakePackage):
 
     homepage = "https://rocm.docs.amd.com/projects/omnitrace/en/latest/index.html"
     git = "https://github.com/ROCm/omnitrace.git"
-    url = "https://github.com/ROCm/omnitrace/archive/refs/tags/rocm-6.2.0.tar.gz"
+    url = "https://github.com/ROCm/omnitrace/archive/refs/tags/rocm-6.0.2.tar.gz"
```

Comment:
What is this url used for? 
If it should be pointing to the latest source tarball, then I think you want: 
<https://github.com/ROCm/rocprofiler-systems/archive/refs/tags/rocm-6.3.0.tar.gz>

Reply:
I've changed the url but it isn't used in the omnitrace recipe. For this recipe we use git instead of the source tarball.